docs/sources/release-notes/release-notes-8-1-3.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Release notes for Grafana 8.1.3"
+[_build]
+list = false
++++
+
+<!-- Auto generated by update changelog github action -->
+
+# Release notes for Grafana 8.1.3
+
+### Features and enhancements
+
+- **Bug:** Return empty plugins.DataTimeSeriesSlice when the frame field vector is empty and does not contain any data. [#38587](https://github.com/grafana/grafana/pull/38587), [@idafurjes](https://github.com/idafurjes)
```

Comment:
Hmm, why is this not included in the changelog.md? Seems like a bug to me? Perhaps something to investigate later...